app/src/test/java/fnsb/macstransit/Helper.java

```diff
@@ -0,0 +1,139 @@
+package fnsb.macstransit;
+
+import android.util.Log;
+
+import org.jetbrains.annotations.Contract;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+
+/**
+ * Created by Spud on 6/27/20 for the project: MACS Transit.
+ * <p>
+ * For the license, view the file titled LICENSE at the root of the project.
+ *
+ * @version 1.0
+ * @since Release 1.2.
+ */
+class Helper {
+
+	/**
+	 * TODO Documentation
```

Comment:
Because these arent production files this is fine, but in the future, these should be filled out